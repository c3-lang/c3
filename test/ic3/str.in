" "
"
"
"0"
"9"
"A"
"Z"
"a"
"z"
"\0"
"\000"
"\001"
"\010"
"\100"
"\x00"
"\x01"
"\x0F"
"\x10"
"\xF0"
"\xFF"
"À"
"É"
"Ÿ"
"à"
"é"
"ÿ"
