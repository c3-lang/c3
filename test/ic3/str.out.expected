" "
"\n"
"0"
"9"
"A"
"Z"
"a"
"z"
"\0"
"\0"
"\001"
"\010"
"\100"
"\0"
"\x01"
"\x0F"
"\x10"
"\xF0"
"\xFF"
"À"
"É"
"Ÿ"
"à"
"é"
"ÿ"
